docs/source/blocks/block-style-wrapper.md

```diff
@@ -62,7 +63,8 @@ The signature for a `schemaEnhancer` is `({schema, formData, intl})`.
 
 ## The `styles` field
 
-The `style` field is mapped to an `objectWidget`, and the `stylesSchema` adds the fields into this field, creating an object that is the sum of all of the widgets assigned to it, and its values.
+The `style` field is mapped to an `objectWidget`.
```

Comment:
The heading above uses `styles` (plural), and the paragraph uses `style` (singular). Please use one or the other for both.